Fix multiple definition error in plt.h
 .
 wl18xx-ti-utils (0.8-1) UNRELEASED; urgency=low
 .
   * Initial release. Closes: #nnnn
     <nnnn is the bug number of your ITP>
Author: root <>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2023-05-26

--- wl18xx-ti-utils-0.8.orig/plt.h
+++ wl18xx-ti-utils-0.8/plt.h
@@ -399,7 +399,7 @@ enum EFUSE_PARAMETER_TYPE_ENMT {
 	TX_BIP_PD_BUFFER_VBIAS_ERROR_E,
 	EFUSE_NUMBER_OF_PARAMETERS_E,
 	EFUSE_LAST_PARAMETER_E = (EFUSE_NUMBER_OF_PARAMETERS_E - 1)
-} EFUSE_PARAMETER_TYPE_ENM;
+};
 
 int get_mac_addr(int ifc_num, unsigned char *mac_addr);
 
